.copilot/config.yml

```
@@ -0,0 +1,4 @@
+repository: company-matching/company-matching-service
+builder:
+  name: paketobuildpacks/builder-jammy-full
+  version: 0.3.414
```

Can we try with the latest builder? I think the recent but not latest ones have some issues with  vulnerabilities in the `libexpat` and `apparmor` packages.

```suggestion
  version: 0.3.431
```